docs/reference/commandline/network_create.md

```diff
@@ -134,7 +135,13 @@ The following are those options and the equivalent docker daemon flags used for
 | `com.docker.network.bridge.enable_icc`           | `--icc`     | Enable or Disable Inter Container Connectivity        |
 | `com.docker.network.bridge.host_binding_ipv4`    | `--ip`      | Default IP when binding container ports               |
 | `com.docker.network.mtu`                         | `--mtu`     | Set the containers network MTU                        |
-| `com.docker.network.enable_ipv6`                 | `--ipv6`    | Enable IPv6 networking                                |
```

Comment:
The context is the paragraph above

>  When creating a custom network, the default network driver (i.e. `bridge`) has additional options that can be passed.  The following are those options and the equivalent docker daemon flags used for docker0 bridge:

`com.docker.network.enable_ipv6` is no longer a valid 'driver option' (also known as 'label') and it will be ignored, so I've removed this line. Instead, `--ipv6` is a valid option for *all* networks.